Fitbit.Portable/Models/SortEnum.cs

```diff
@@ -7,8 +7,8 @@ namespace Fitbit.Api.Portable.Models
 {
     public enum SortEnum
     {
-        Asc,
```

Comment:
Why have these been made lowercase? This is not standard in naming of enum values, nor are the values used as the string representation.

Reply:
I am not really sure why i did this, will change it back.